lib/plugins/features/movies/controller.js

```diff
@@ -32,3 +35,43 @@ exports.getAllMovies = (params) => {
     }
   }).fetchAll();
 };
+
+exports.createLocation = async (movieId, payload) => {
+  const movie = await new Movie({ id: movieId })
+    .fetch({
+      withRelated: 'locations',
+      require: true
+    })
+    .catch(() => {
+      throw Boom.notFound('Movie not found');
```

Comment:
Added this as some folks in the #eng-delivery-engine channel suggested using `Boom` for error handling.